pom.xml

```
@@ -3,34 +3,45 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>3.55</version>
+    <version>4.19</version>
```

Use the newest parent pom
```suggestion
    <version>4.24</version>
```